source/common/upstream/load_balancer_impl.cc

```diff
@@ -775,6 +845,24 @@ void EdfLoadBalancerBase::refresh(uint32_t priority) {
   }
 }
 
+bool EdfLoadBalancerBase::noHostsAreInSlowStart() {
```

Comment:
ptal @mattklein123 @snowp

Reply:
I think this iml makes sense to me at a high level, and avoids complex timers around when hosts end slow start.

cc @antoniovicente @tonya11en who might also be interested in taking a look and brainstorming other options.